### PR TITLE
Add sample data and sternberg stylesheet.

### PR DESCRIPTION
**Pull request by markpbaggett**
**GitHub Issue**: [DLTN_XSLT-127](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/127)

* [Mapping](https://dltn-technical-docs.readthedocs.io/en/latest/mappings/rhodes.html#col-10267-11338-irma-o-sternberg-overton-park-expressway-collection)

## What does this Pull Request do?

Transforms Sternberg XOAI to MODS.

## What's new?

1. An example feed.
2. A set-based stylesheet.

## How should this be tested?

Run the stylesheet versus the sample record.  Is it valid?  Does it match the mapping?

## Additional Notes:

1. We can git rm the stylesheet once the master is approved and merged.
2. There are 181 bad records that will have to be dealt with by check_and_harvest.

## Interested parties

@mlhale7

### CODE REVIEW

File: Sample_Data/Rhodes/sternberg.xml [hunk 1]
```diff
@@ -0,0 +1,16766 @@
...
+</element>
+</element>
+</element>
+<element name="bundle">
+<field name="name">TEXT</field>
+<element name="bitstreams">
+<element name="bitstream">
+<field name="name">OPE-19_008.pdf.txt</field>
+<field name="originalName">OPE-19_008.pdf.txt</field>
+<field name="description
```
Comment (mlhale7) [inline Sample_Data/Rhodes/sternberg.xml]: Here's an example of bad data . . .

**Comment by mlhale7** [comment]
@markpbaggett - Is it possible to add only the records we are going to add to the server as the sample data so that this can be tested more effectively?

**Comment by markpbaggett** [comment]
Uh, I guess we could, but that's a lot more work, and you'd need to do that for Farnsworth as well. 

I don't see how that makes the test more effective though.  If you're removing the problems, doesn't that make a test pass when it should fail?

Best,
Mark

**Comment by mlhale7** [comment]
@markpbaggett - I might be misunderstanding, but I believe we are never going to run the transform against records that fail (don't have thumbnails etc.) anyway. Why would we test a transform against records it won't be seeing? We'll add the records that pass to the server and then get these to Repox and transform them?

**Comment by markpbaggett** [comment]
Okay, all records missing thumbnails have been removed.

**Comment by mlhale7** [comment]
I was hopeful that only those items without thumbnails had the crappy metadata I was seeing before. We still have two records with "UPDATE THIS INFORMATION" in the abstract. While not very streamlined for other new sets, would it be possible to add an exclusion for these records (like 
```<xsl:choose><xsl:when test=".!='UPDATE THIS INFORMATION'"><abstract><xsl:apply-templates/></abstract></xsl:when></xsl:choose>```? We can manually remove them as well, but we'll have to do this any time we update this set then.

Again, the transform works great, but we don't want to share this data as it is with DPLA. Suggestions?

**Comment by markpbaggett** [comment]
It'd be nice if they could fix this, but that will work for this instance.

**Pull request merged**